Sources/Utilities/Storage/Session/PersistentSessionManager.swift

```diff
@@ -148,13 +148,17 @@ final class PersistentSessionManager: SessionManager {
             .readItem(itemName: .accessToken)
         tokenProvider.update(accessToken: accessToken)
     }
+    
+    func stopCurrentSession() {
```

Comment:
interesting, why "stop" rather than "end"? 🙂

Reply:
Just because the method below is "end" 🙂

Reply:
Thinking about this some more, this isn't right as it doesn't _always_ prompt for local auth based on `sessionExists` in the `SceneDelegate` if you call this method when sending to the background.. 🤔